docs/_docs/Users/build.md

```diff
@@ -5,6 +5,95 @@ order: 1
 toc: true
 ---
 
+## Ubuntu 18.04 and Linux Mint 19
+
+
+* Save an empty JKS file with the default 'changeit' password for Java cacerts.
+
+`sudo /usr/bin/printf '\xfe\xed\xfe\xed\x00\x00\x00\x02\x00\x00\x00\x00\xe2\x68\x6e\x45\xfb\x43\xdf\xa4\xd9\x92\xdd\x41\xce\xb6\xb2\x1c\x63\x30\xd7\x92' > /etc/ssl/certs/java/cacerts`
+
+* Re-add all the CA certs into the previously empty file.
+
+`sudo /var/lib/dpkg/info/ca-certificates-java.postinst configure`
+
+* Add the Wine repository
+
+`sudo apt-add-repository https://dl.winehq.org/wine-builds/ubuntu/`
+
+`wget -nc https://dl.winehq.org/wine-builds/Release.key && sudo apt-key add Release.key`
+
+* Remove exising version of wine
+
+`apt-get purge wine* libwine`
+
+* Update the repository information
+
+`sudo apt-get update`
+
+* Install wine packages (This is needed to stop the empty dialog boxes)
+
+`sudo apt-get install --install-recommends winehq-devel`
+ 
+
+* Install packages
+
+`sudo apt-get -y install git maven openjdk-8-jdk openjfx libxext6:i386`
+
+* Change to build directory
+
+`mkdir -P ~/build`
+
+`cd ~/build`
+
+* Grab the source
+
+`git clone https://github.com/PhoenicisOrg/phoenicis.git`
+
+* Go in the source directtory
+
+`cd phoenicis`
+
+* Change the default version of java from 10 to java 8
+
+`export JAVA_HOME="/usr/lib/jvm/java-8-openjdk-amd64"`
+
+`sudo update-java-alternatives --set /usr/lib/jvm/java-1.8.0-openjdk-amd64`
+
+* Build
+
+`mvn clean package`
+
+* Unzip the distribution
+
+`cd phoenicis-dist/target`
+
+`unzip phoenicis-dist.zip`
+
+* Clean up the existing config
+
+`rm -R ~/.Phoenicis`
+
+* Run! This is the only line you need from now on.
+
+`~/build/phoenicis/phoenicis-dist/target/phoenicis-dist/phoenicis.sh`
+
+There is an issue with the latest steam client not downloading. See https://bugs.winehq.org/show_bug.cgi?id=45329
```

Comment:
These Steam instructions do not belong here (this is only about building Phoenicis). We should consider updating [SteamScript](https://github.com/PhoenicisOrg/scripts/blob/master/Engines/Wine/QuickScript/Steam%20Script/script.js) instead.

Reply:
@madoar @qparis what do you think?